cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -131,3 +131,11 @@
 default[:bcpc][:hadoop][:os][:group][:hadoop][:members]=["hdfs","yarn"]
 default[:bcpc][:hadoop][:os][:group][:hdfs][:members]=["hdfs"]
 default[:bcpc][:hadoop][:os][:group][:mapred][:members]=["yarn"]
+
+default[:bcpc][:hadoop][:hdfs][:ldap][:integration] = false
+default[:bcpc][:hadoop][:hdfs][:ldap][:user] = "" #must be LDAP DN
+default[:bcpc][:hadoop][:hdfs][:ldap][:domain] = "BCPC.EXAMPLE.COM"
+default[:bcpc][:hadoop][:hdfs][:ldap][:port] = 389
+default[:bcpc][:hadoop][:hdfs][:ldap][:password] =  nil
+default[:bcpc][:hadoop][:hdfs][:ldap][:search][:filter][:user]="(&amp;(objectclass=user)(sAMAccountName={0}))"
```

Comment:
Can this use Ruby's [string.encode](http://ruby-doc.org/core-1.9.3/String.html#method-i-encode) with the `:xml` type to encode this in the attribute rather than requiring the human to do the encoding?